core/src/chiplets/hasher.rs

```diff
@@ -31,6 +31,12 @@ pub type HasherState = [Felt; STATE_WIDTH];
 /// permutation.
 pub const STATE_WIDTH: usize = Hasher::STATE_WIDTH;
 
+/// Index of the column holding row addresses in the trace.
+pub const ROW_COL_IDX: usize = NUM_SELECTORS;
+
+/// The hasher state portion of the execution trace, located in 4 .. 16 columns.
+pub const STATE_COL_RANGE: Range<usize> = create_range(ROW_COL_IDX + 1, STATE_WIDTH);
+
```

Comment:
We should redefine the constants one level up in `mod` to use these ones, following the same pattern used for `bitwise` constants. This will ensure a single source of truth regardless of whether we're accessing only within the hasher trace or within the full trace

Reply:
Sorry, fixed.